taxonomy-editor-frontend/src/pages/allentries/index.jsx

```diff
@@ -38,7 +50,7 @@ const Entry = () => {
     function handleAddNode() {

```

Comment:
for these and others, let's try to stick to `const handleAddNode = () => {...}` we are mixing both synthax

Reply:
Yep, I refactored the code to use this syntax everywhere